mongonaut/forms/form_mixins.py

```diff
@@ -70,30 +64,22 @@ def __init__(self, model, instance=None, form_post_data=None):
         self.form = forms.Form()
 
         if not isinstance(self.model, TopLevelDocumentMetaclass):
-            raise TypeError(u"The model supplied must be a mongoengine Document")
+            raise TypeError("The model supplied must be a mongoengine Document")
 
         if self.is_initialized and not isinstance(self.model_instance, self.model):
-            raise TypeError(u"The provided instance must be an instance of the given model")
+            raise TypeError("The provided instance must be an instance of the given model")
 
         if self.post_data_dict is not None and not isinstance(self.post_data_dict, dict):
-            raise TypeError(u"You must pass in a dictionary for form_post_data")
+            raise TypeError("You must pass in a dictionary for form_post_data")
 
     def get_form_field_dict(self, model_dict):
         """
         Takes a model dictionary representation and creates a dictionary
         keyed by form field.  Each value is a  keyed 4 tuple of:
         (widget, mode_field_instance, model_field_type, field_key)
         """
-        return_dict = OrderedDict()
-        # Workaround: mongoengine doesn't preserve form fields ordering from metaclass __new__
```

Comment:
Did this change? If not, `OrderDict` should still be used, unless this project plans to forgo all Python 2 support. I'm also not certain that all versions of Python 3.X support a default ordering of keys/values. I think it is predictable, but not necessarily always the same order. 

What motivates this change? The fact that [OrderDict](https://docs.python.org/3/library/collections.html#collections.OrderedDict) still exists makes be think that this code will not always behave the same.